src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -67,6 +67,14 @@ using namespace mongo;
 
 /* ****************************************************************************
 *
+* stringFilterP - 
+*/
+extern __thread StringFilter* stringFilterP;
```

Comment:
These '__thread StringFilter* stringFilterP' are part of this PR.
Due to leakage (depending on the thread mechanism of MHD), detected in the coming PR, this variable is no longer a thread variable but a part of ConnectionInfo, but that is for the next PR, not this one.

NTC